xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/service/BittrexAccountService.java

```diff
@@ -93,4 +94,15 @@ public TradeHistoryParams createFundingHistoryParams() {
       throw BittrexErrorAdapter.adapt(e);
     }
   }
+
+  @Override
+  public String withdrawFunds(Currency currency, BigDecimal amount, String address)
+      throws IOException {
+    try {
+      BittrexWithdrawal createdWithdrawal = createNewWithdrawal(currency, amount, address);
+      return createdWithdrawal.getId();
```

Comment:
can you just inline this? instead of having two lines?

Reply:
Sure, done.